lava-v2-jobs-from-api.py

```diff
@@ -143,19 +143,26 @@ def get_job_params(config, template, opts, device, build, defconfig, plan):
 
     nfsrootfs_url = None
     initrd_url = None
+    rootfs_prompt = "\(initramfs\)"
+
     if 'kselftest' in plan:
         initrd_url = KSELFTEST_INITRD_URL.format(initrd_arch)
     else:
         initrd_url = INITRD_URL.format(initrd_arch)
     if 'nfs' in plan:
         nfsrootfs_url = NFSROOTFS_URL.format(initrd_arch)
         initrd_url = None
+        rootfs_prompt = "root@debian-(.*):~#"
     if build['modules']:
         modules_url = urlparse.urljoin(
             storage, '/'.join([url_px, build['modules']]))
     else:
         modules_url = None
 
+    # hack for compatibility
+    if (initrd_url and 'buildroot' in initrd_url) or (nfsrootfs_url and 'buildroot' in nfsrootfs_url):
```

Comment:
It would be nicer to have a data structure with the various rootfs variants, which plans they're compatible with and which prompts they use.  Some kind of rootfs map essentially.